cinema/serializers.py

```diff
@@ -1 +1,81 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import Actor, CinemaHall, Genre, Movie, MovieSession
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    full_name = serializers.CharField(source="__str__", read_only=True)
+
+    class Meta:
+        model = Actor
+        fields = ("id", "first_name", "last_name", "full_name")
+
+    def get_full_name(self, obj):
+        return str(obj)
```

Comment:
why do u need this ?

Reply:
I saw in one tutorial about the definition of the get_<field_name> method, and you are right, it is not used here